libs/langchain/langchain/storage/in_memory.py

```diff
@@ -0,0 +1,79 @@
+"""In memory store that is not thread safe and has no eviction policy.
+
+This is a simple implementation of the AbstractStore using a dictionary that is useful
+primarily for unit testing purposes.
+"""
+from typing import Any, Dict, Iterator, List, Optional, Sequence, Tuple
+
+from langchain.schema import BaseStore
+
+
+class InMemoryStore(BaseStore[str, Any]):
+    """In-memory implementation of the AbstractStore using a dictionary.
+
+    Examples:
+        ... code-block:: python
+
+            store = InMemoryStore()
+            store.mset([('key1', 'value1'), ('key2', 'value2')])
+            store.mget(['key1', 'key2'])
+            ['value1', 'value2']
```

Comment:
would add # (ik this snippet would still run but it's intended to just indicate output i presume)